libraries/AP_HAL_ChibiOS/hwdef/Aocoda-RC-F405/hwdef-bl.dat

```diff
@@ -0,0 +1,38 @@
+# hw definition file for processing by chibios_pins.py
+# for speedybeef4 bootloader
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID. See Tools/AP_Bootloader/board_types.txt
+APJ_BOARD_ID AP_HW_AOCODA-RC-F405V3
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 1024
+
+# don't allow bootloader to use more than 16k
+FLASH_USE_MAX_KB 16
```

Comment:
So you are constraining this to 16k but then further down you allow 64k - why the difference